Documentation/architecture/cross-platform-cryptography.md

```diff
@@ -48,6 +42,44 @@ This would result in a `PlatformNotSupportedException` when invoking the `Create
 In the future there is a possibility that new cipher/chaining modes may be added to .NET Core and that these new modes may not apply to all symmetric algorithms.
 This would likely result in a `NotSupportedException` when using the set-accessor of the `Mode` property on the `SymmetricAlgorithm` object, but this prediction is subject to change.
 
+## Authenticated Encryption
+
+Authenticated Encryption (AE) support is provided for AES-CCM and AES-GCM via `System.Security.Cryptography.AesCcm` and `System.Security.Cryptography.AesGcm`, respectively.
+
+On macOS the AesCcm and AesGcm classes use OpenSSL for support.
+Users on macOS need to obtain an appropriate copy of OpenSSL (libcrypto) for these types to function, and it must be in a path that the system would load a library from by default.
+Obtaining OpenSSL from a package manager, such as Homebrew, is recommended (but not required).
+The libcrypto.0.9.7.dylib and libcrypto.0.9.8.dylib libraries included in macOS are from older versions of OpenSSL, and will not be used.
+The libcrypto.35.dylib, libcrypto.41.dylib, and libcrypto.42.dylib are LibreSSL, and will not be used.
+
```

Comment:
Should we explicitly call out that on other platforms it's performed by the system libraries?